### PR TITLE
Docs for XPENDING IDLE

### PR DESCRIPTION
**Pull request by guybe7**

### CODE REVIEW

File: commands/xpending.md [hunk 1]
```diff
@@ -103,10 +103,38 @@ we have a pending entries list data structure both globally, and for
...
 a single consumer.
 
+If the owner argument is an empty string, the command will return entries
+assigned to any consumer:
+
+```
+> XPENDING mystream group55 - + 10 ""
+```
+
+The above case will return the 10 (or less) first PEL entries of the entire group
```
Comment (itamarhaber) [inline commands/xpending.md:113]:
```suggestion
The above case will return the first 10 (or less) PEL entries of the entire group
```

File: commands/xpending.md [hunk 2]
```diff
@@ -103,10 +103,38 @@ we have a pending entries list data structure both globally, and for
...
+
+Since version 6.2 it is possible to filter entries by their idle-time,
+given in milliseconds (Useful if you want to use `XCLAIM` to claim entries
+that have not been processed for some amount of time):
```
Comment (itamarhaber) [inline commands/xpending.md:117]:
```suggestion
given in milliseconds (useful for `XCLAIM`ing entries that have not been
processed for some time):
```

File: commands/xpending.md [hunk 3]
```diff
@@ -103,10 +103,38 @@ we have a pending entries list data structure both globally, and for
...
+
+```
+> XPENDING mystream group55 - + 10 "" IDLE 9000
+> XPENDING mystream group55 - + 10 consumer-123 IDLE 9000
+```
+
+The first case will return the 10 (or less) first PEL entries of the entire group
+that are idle for over 9 seconds.
+The second case will return the 10 (or less) first PEL entries of `consumer-123`
+that are idle for over 9 seconds.
```
Comment (itamarhaber) [inline commands/xpending.md:127]:
```suggestion
The first case will return the first 10 (or less) PEL entries of the entire group
that are idle for over 9 seconds, whereas in the second case only those of
`consumer-123`.
```

**Review by itamarhaber — COMMENTED** [review]
Looks good - minor edits.

**Comment by guybe7** [comment]
@itamarhaber done.

**Pull request merged**